src/test/scala/tech/cryptonomic/conseil/tezos/DatabaseConversionsTest.scala

```diff
@@ -776,14 +776,17 @@ class DatabaseConversionsTest
 
       "convert a Proposals operation to a database row" in {
 
-        val converted = (block, groupHash, Proposals: Operation).convertTo[Tables.OperationsRow]
+        val converted = (block, groupHash, sampleProposals: Operation).convertTo[Tables.OperationsRow]
 
         converted.operationId shouldBe 0
         converted.operationGroupHash shouldBe groupHash.value
         converted.blockHash shouldBe block.data.hash.value
         converted.blockLevel shouldBe block.data.header.level
         converted.timestamp shouldBe Timestamp.from(block.data.header.timestamp.toInstant)
         converted.kind shouldBe "proposals"
+        converted.source shouldBe Some(ContractId("tz1VceyYUpq1gk5dtp6jXQRtCtY8hm5DKt72"))
+        //converted.period shouldBe Some(10)
+        converted.proposal shouldBe Some("[\"Psd1ynUBhMZAeajwcZJAeq5NrxorM6UCU4GJqxZ7Bx2e9vUWB6z\"]")
 
         forAll(
```

Comment:
As can be seen here, every test on operation conversions additionally takes care that every field in the row that is unused by a specific operation kind, is verified to be empty.

This means that newly introduced fields (like `period`) will need to be added to this "empty field list" in the test, for every operation type that doesn't actually use the field

Reply:
Yeah, I think uncommenting it out would solve the problem *sweatdrop my bad.

Reply:
Did you reply to the wrong question?